environment.yml

```diff
@@ -13,3 +13,7 @@ dependencies:
   # TODO nf-core: Add required software dependencies here
   - bioconda::fastqc=0.11.8
   - bioconda::multiqc=1.7
```

Comment:
```suggestion
```
As neither FastQC, nor MultiQC will be used in `sim-seqer` tool, I think you can safely remove those dependencies from `environment.yml` file

Reply:
Removed